package.json

```diff
@@ -23,6 +23,7 @@
     "chokidar": "^1.4.2",
     "clone": "^2.0.0",
     "compression": "^1.6.0",
+    "convict": "^4.0.0",
```

Comment:
Can `convict` remove `is-my-json-valid` from here?
https://github.com/nodecg/nodecg/blob/037709a9f78403b2d7a8e76b9487b48033fb4f14/lib/replicant/shared.js#L241
Looks to be the last usage in the project.

Same with `json-schema-defaults`
https://github.com/nodecg/nodecg/blob/775c15824b0ec4b4138c4a8677179549b0608fd5/lib/replicant/replicant.js#L113-L119

Reply:
No, it cannot. Convict uses its own special schema format, which is not compatible (nor does it have the same goals/purpose) as JSON Schemas, which are used in replicants. Thank you for checking thought! The extra eyes looking for places to remove unnecessary packages are very appreciated.

Reply:
Yep, wasn't entirely sure if it was a straight replacement for all schemas.